composer.json

```diff
@@ -16,11 +16,15 @@
 		"issues": "https://github.com/Carrooi/Nette-Menu/issues"
 	},
 	"require": {
-		"php": ">=7.1",
+		"nette/application": "^2.4",
```

Comment:
Do we need all these new dependencies? And if yes, shoudn't they be in the `requireDev` section?

Reply:
Maybe not, removing ...